emails/templates/missed-vote.js

```diff
@@ -48,8 +48,7 @@ module.exports = function() {
     mockData: {
       ...accountData('0xef0f7ecef8385483ac8a2e92d761f571c4b782bd'),
       date: 'Thursday, 17 Dec. 2019',
-      lockedAnjBalanceUrl: '',
-      learnMoreUrl: '',
+      lockedAnjBalanceUrl: 'https://court.aragon.org/dashboard'
```

Comment:
We should parameterize this URL as well

Reply:
It is, this is just used for generating mock templates for preview (you can run `yarn build:mock` in email folder)